Training/2.0 Libraries.md

```diff
@@ -59,7 +65,10 @@ Normally we would want to be able to extract the information from tables directl
 Using your DataFrame from Task 4a), calculate the average systolic blood pressure of Dementia patients.
 
 ```
-
+import pandas as pd
+df=pd.read_csv('M:\Training2_example.csv')
+mean_series=df.mean(numeric_only=True)
+mean_sysp=mean_series['Systolic Blood Pressure']
```

Comment:
Good attempt, but this will calculate the average pressure of all patients, not just the Dementia ones! Give it another go, if you can't figure it out, I will explain tommorrow.